Library/Formula/dub.rb

```diff
@@ -10,6 +10,12 @@ class Dub < Formula
     sha256 "4afe785f25db496f8634c19211d68c7ea05778f9ed1e7e772f8aa9ac701beca1" => :mountain_lion
   end
 
+  devel do
+      url "https://github.com/D-Programming-Language/dub/archive/v0.9.24-rc.2.tar.gz"
+      version "0.9.24-rc2"
+      sha1 "43fbdbca2313b997b219c50e13ff43742ae0d302"
```

Comment:
done. I presume this is the new standard and I should do this for all the formulae I maintain?